lib/Backend/CodeGenWorkItem.h

```diff
@@ -42,7 +42,7 @@ struct CodeGenWorkItem : public JsUtil::Job
 
     uint GetFunctionNumber() const
     {
-        return this->jitData.jitData->bodyData->funcNumber;
+        return this->functionBody->GetFunctionNumber();
```

Comment:
this->jitData.jitData is not initialized until NativeCodeGenerator::CodeGen, whereas functionBody is passed as argument to the constructor of CodeGenWorkItem so is there from the beginning.

This area definitely got a bit messy with OOP JIT, and I'm planning on doing bigger cleanup of CodeGenWorkItem to address this sort of strangeness.